bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -158,6 +158,9 @@ public void initialize() {
                 updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.CONFIGURATION_PENDING,
                         messages.get("status.unknown.initializing"));
                 start = initializeThing();
+
+                // Promote Shelly Manager usage
+                logger.info("{}: {}", thingName, messages.get("status.managerstarted", localIP, localPort));
```

Comment:
Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. This could be debug. See https://next.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
Many people don't recognize the Shelly Manager UI, because OH doesn't provide an interface for UI extensions provided by the binding (as already requested a long time ago) and many don't reads the manual (RTFM). Therefore I'm adding this info to the log to promote this useful feature.